src/components/pages/datasets/pep/PepDetail.jsx

```diff
@@ -0,0 +1,226 @@
+                    <span className="italic">{i18n.language === 'uk' ? person.from_person_relationship_type : person.from_person_relationship_type_en} — </span>
+                    <span className="capitalize underline">{i18n.language === 'uk' ? person.from_person.fullname : person.from_person.fullname_en}</span>
+                  </li>
+                )) : null}
+              </ul>
+              {fromPersonLinksLength + toPersonLinksLength > 3 ? (
+                <div className="flex flex-row cursor-pointer text-blue-800" onClick={() => toggleOpenRelatedPersons()}>
+                  {openRelatedPersons ? [<ChevronUp className="w-4 h-6" />, t('viewLess')] : [<ChevronDown className="w-4 h-6" />, t('viewMore')]}
+                </div>
+              ) : null}
+            </div>
+          </div>
+          <div className="pl-5 flex flex-row">
+            <div className="w-64 font-medium">{t('pepDetail.relatedCompanies')}:</div>
+            <div className="max-w-xl">
+              <ul className={`overflow-y-hidden list-disc list-inside ${openRelatedCompanies ? '' : 'max-h-3'}`}>
+                {data.related_companies && data.related_companies.length > 0 ?
+                  data.related_companies.map((company) => (
+                    <li>
+                      <span className="underline">
+                        {i18n.language === 'uk' ? company.company.name : company.company.name_en} ({company.company.edrpou})
+                      </span>
+                    </li>
+                  )) : '---'}
+              </ul>
+              {data.related_companies && data.related_companies.length > 3 ? (
+                <div className="flex flex-row cursor-pointer text-blue-800" onClick={() => toggleOpenRelatedCompanies()}>
+                  {openRelatedCompanies ? [<ChevronUp className="w-4 h-6" />, t('viewLess')] : [<ChevronDown className="w-4 h-6" />, t('viewMore')]}
+                </div>
+              ) : null}
+            </div>
+          </div>
+          <div className="pl-5 flex flex-row">
+            <div className="w-64 font-medium">{t('pepDetail.sanctions')}:</div>
+            <div className="max-w-xl">{data.sanctions ? data.sanctions : '---'}</div>
+          </div>
+          <div className="pl-5 flex flex-row">
+            <div className="w-64 font-medium">{t('pepDetail.criminalRecords')}:</div>
+            <div className="max-w-xl">{data.criminal_record ? data.criminal_record : '---'}</div>
+          </div>
+          <div className="pl-5 flex flex-row">
+            <div className="w-64 font-medium">{t('pepDetail.criminalProceedings')}:</div>
+            <div className="max-w-xl">{
+              (() => {
+                if (data.criminal_proceedings) {
+                  const proceedings = data.criminal_proceedings.replace(/(\\r\\n)+/g, '</br>').replace(/(\\")+/g, '"');
+                  return openCriminalProceedings ?
+                    <div dangerouslySetInnerHTML={{ __html: proceedings }} /> :
```

Comment:
really? you use `dangerouslySetInnerHTML`?

Reply:
it fast solution

Reply:
no javascript in jsx

Reply:
done